docs/source/dashboard.rst

```diff
@@ -0,0 +1,250 @@
+        <tr>
+            <td>
+                <div role="img" aria-label="red square" style="color:rgba(255, 0, 0, 1); font-size: 25px ">&#9632;</div>
+            </td>
+            <td>Idle: Does not have enough work to stay busy.</td>
+        </tr>
+    </table>
+
+In this plot on the dashboard we have two extra tabs with the following information:
+
+**CPU Utilization**
+
+The *CPU* tab shows the cpu usage per-worker as reported by ``psutils`` metrics. 
+
+**Occupancy**
+
+The *Occupancy* tab shows the occupancy, in time, per worker. The total occupancy for a worker is the total expected runtime
+for all tasks currently on a worker. For example, an occupancy of 10s means an occupancy of 10s means that the worker 
+estimates it will take 10s to execute all the tasks it has currently been assigned.
+
+.. _dashboard.task-stream:
+
+Task Stream
+-----------
+
+The task stream is a view of all the tasks across worker-threads. Each row represents a thread and each rectangle represents 
+an individual tasks. The color for each rectangle corresponds to the task-prefix of the task being performed and it matches the color 
+of the *Progress* plot (see Progress section). This means that all the individual tasks part of the `inc` task-prefix for example, will have 
+the same randomly assigned color from the viridis color map. 
+
+There are certain colors that are reserved for a specific kinds of tasks:
+
+.. raw:: html
+
+    <table>
+        <tr>
+            <td><b>Color</b></td><td><b>Meaning</b></td>
+        </tr>
+        <tr>
+            <td>
+                <div role="img" aria-label="light red square" style="color:rgba(255, 0, 0, 0.4); font-size: 25px ">&#9632;</div>
+            </td>
+            <td>Transferring data between workers tasks.</td>
+        </tr>
+        <tr>
+            <td>
+                <div role="img" aria-label="light orange square" style="color: rgba(255,165,0, 0.4); font-size: 25px ">&#9632;</div>
+            </td>
+            <td>Reading from or writing to disk.</td>
+        </tr>
+        <tr>
+            <td>
+                <div role="img" aria-label="light grey square" style="color:rgba(128,128,128, 0.4); font-size: 25px ">&#9632;</div>
+            </td>
+            <td>Serializing/deserializing data.</td>
+        </tr>
+        <tr>
+            <td>
+                <div role="img" aria-label="black square" style="color:rgba(0, 0, 0, 1); font-size: 25px ">&#9632;</div>
+            </td>
+            <td>Erred tasks.</td>
+        </tr>
+    </table>
+
+
+.. figure:: images/dashboard_taskstream_healthy.png
+    :alt: Task Stream pane showing eight horizontal bars, one per worker-thread, with different shades of 
+    greens and blues and some red. 
+
+.. figure:: images/dashboard_task_stream_unhealthy.png
+    :alt:  Task Stream pane showing twelve horizontal bars, one per worker-thread, with majority of white space
+    in each bar, and displaying some purple, red and orange rectangles.   
```

Comment:
```suggestion
    :alt: An example of a poorly optimized Task Stream, the bar chart shows a lot of white space and red rectangles.
```

Reply:
SImilar comment than for the other plot, I'm not sure about "poorly optimized Task Stream"